cstr/src/lib.rs

```diff
@@ -89,6 +89,26 @@ pub unsafe fn parse_attribute_string(text: *const WideChar) -> String {
     from_widechar_vec_lossy(dst).replace(char::from(0), ";")
 }
 
+///
+/// parse_attribute_string converts a null-separted u16 doubly-null terminated cstring to a Rust
```

Comment:
```suggestion
/// parse_attribute_string_a converts a null-separted u16 doubly-null terminated cstring to a Rust
```
Should we include the suffix in this function comment, and the `_w` for the previous function? And should we distinguish their behaviors in the comments as well? Right now, the comments for `parse_attribute_string_w` and `parse_attribute_string_a` are identical, which feels incorrect.

Reply:
Yes, absolutely. I actually changed these in #113 , see [here](https://github.com/mongodb/mongo-odbc-driver/pull/113/files#diff-2f5e9a143c4e8edf9d00c1f6589095b53467dbf69f8e1438957e8c3ed7671f6eR71-R79) and [here](https://github.com/mongodb/mongo-odbc-driver/pull/113/files#diff-2f5e9a143c4e8edf9d00c1f6589095b53467dbf69f8e1438957e8c3ed7671f6eR95-R103).

I can also change it here and deal with the small merge conflict.

Reply:
Oh thanks for the links. Looks good 👍